src/Home.js

```diff
@@ -47,12 +47,23 @@ export default function Home() {
     setTodoDescription("");
   }
 
+  async function deleteTodo(todo) {
+    try {
+      await DataStore.delete(todo);
+    } catch (e) {
+      console.log('Delete failed: $e');
```

Comment:
Looks like you are trying to use a template literal here? Check the documentation [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals), your syntax is a bit off

Reply:
Thanks for the documentation link. I have updated the template literal string.